src/GooglePay.ts

```diff
@@ -40,27 +46,28 @@ if (Platform.OS === 'android') {
   googlePayEmitter.addListener('onGooglePayCanceled', onNativeGooglePayCanceled);
 }
 
-async function initializeGooglePay(squareLocationId, environment) {
+const initializeGooglePay = async (squareLocationId:string, environment:any) => {
```

Comment:
Should environment an `integer`?

Reply:
Yes. Corrected it as the environment has two values to associate itself with.
EnvironmentProduction = 1;
EnvironmentTest = 3;